server/py/services/api/tests/unit/utils/projects/test_follower_member.py

```diff
@@ -468,7 +468,7 @@ async def test_list_project_summaries_fails_to_list_pipeline_runs(
     )
     framework.utils.singletons.db.get_db().get_project_resources_counters = (
         unittest.mock.AsyncMock(
-            return_value=tuple({project_name: i} for i in range(12))
+            return_value=tuple({project_name: i} for i in range(14))
```

Comment:
What does this magic number mean?
It seems to me it should depend on the `ProjectSummary` class `"<>_count"` properties, but the number doesn't quite fit.
If it is what I think it is, maybe we can calculate it dynamically so it will be resilient to future changes. E.g:
```
num_count_properties = sum(1 for field_name in schemas.ProjectSummary.__fields__ if field_name.endswith('_count'))
```